Scorecounter/app/src/main/res/layout/activity_select_format.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/cricket">
+
+    <LinearLayout
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_gravity="center_vertical"
+        android:orientation="vertical">
+
+        <TextView
+            style="@style/headings"
+            android:text="Select match format."
+            android:textColor="@color/colorBlack"
+            android:layout_marginLeft="8dp"
+            android:layout_marginRight="8dp"
+            android:layout_marginBottom="20dp"/>
+
+        <Button
+            android:id="@+id/T20"
+            style="@style/button"
+            android:textSize="36sp"
+            android:text="T-20"
+            android:background="@drawable/rounded_button_green"
+            android:padding="8dp"
+            android:layout_margin="20dp"/>
+
+        <Button
+            android:id="@+id/ODI"
+            style="@style/button"
+            android:text="ODI"
+            android:textSize="36sp"
+            android:background="@drawable/rounded_button_green"
+            android:padding="8dp"
+            android:layout_margin="20dp"/>
```

Comment:
Also, don't use hardcoded values for dimensions.
Use resource files instead

Reply:
done